README.md

```diff
@@ -14,54 +22,60 @@ For documentation on previous versions of the Molecular Targets Platform, please
 
 #### Contents
 - [Datasets](#datasets)
-- [Somatic Alterations](#Somatic_Alterations)
-- [Gene Expression](#Gene_Expression)
+- [DNA Sequencing](#DNA_Sequencing)
+- [RNA Sequencing](#RNA_Sequencing)
+- [Methylation](#Methylation)
 - [Data Availability](#Data_Availability)
 
 ## Datasets
 
-While adult pan-cancer repositories have existed and accelerated cancer research for a decade, pediatrics cancers have been excluded, despite having different genetic and molecular etiologies than adult cancers. Over the past few years larger pediatric consortia, both disease-specific and pan-cancer, have tried to address this disparity. The Molecular Targets Platform is harmonizing the data from across these different consortia in one unified location where it can be queried for associations between putative targets and pediatric cancers. As the project is ongoing, more data will continue to be added, but this current release includes 3 pediatric consortia datasets as well as GTEx data for comparisons to normal tissue expression:
+While adult pan-cancer repositories have existed and accelerated cancer research for a decade, pediatrics cancers have been excluded, despite having different genetic and molecular etiologies than adult cancers. Over the past few years larger pediatric consortia, both disease-specific and pan-cancer, have tried to address this disparity. The Molecular Targets Platform is harmonizing the data from across these different consortia in one unified location where it can be queried for associations between putative targets and pediatric cancers. As the project is ongoing, more data will continue to be added, but this current release includes **XXX** pediatric consortia datasets as well as GTEx data for comparisons to normal tissue expression:
 
 | Dataset | Number of Samples |
 | --- | --- |
-| PBTA | 2,984 |
-| TARGET | 4,489 |
-| GMKF | 884 |
+| PBTA | 4,164 |
+| TARGET | 5,252 |
+| GMKF | 881 |
+| DGD | 1,799 |
 | GTEx | 17,382 |
-| TCGA | |
+| TCGA | 10,414 |
```

Comment:
We might want to either name this as number of biospecimens or number of tumor samples ("samples" alone is vague) and re-calculate (also adding code for the calculation).

Reply:
This is an update that needs to be made to main then actually. Maybe I should write another ticket to quickly update the sample tables there?

Reply:
we have data in OPC v11 but only a subset of that data will be in MTP v1.1 - i.e we don't plan to have TCGA data yet. so we may want to exclude doc/stats on data/cohorts that are not being displayed/available via the MTP v1.1 (even though they are part of the OPC v11 releases)

Reply:
I would like to leave it for now. Since this is going to the dev branch and won't be public-facing, I would rather have more information rather than less and delete when MTPv1.1 is going to go live since it's not completely settled what is going to be in the release yet

Reply:
>This is an update that needs to be made to main then actually. Maybe I should write another ticket to quickly update the sample tables there?

This sounds like a plan, and not urgent

Reply:
Ok then, I will leave this as is in dev for the moment and put in a ticket